docs/internals/contributing/writing-code/unit-tests.txt

```diff
@@ -28,7 +28,7 @@ Running the tests requires a Django settings module that defines the
 databases to use. To make it easy to get started, Django provides and uses a
 sample settings module that uses the SQLite database. To run the tests:
 
-.. code-block:: bash
+.. code-block:: console
```

Comment:
Likely not. I expect Python to be default language and thus use `::` for that. If a file has a huge majority for another language, I'd be okay with setting the default highlight to that language.